cinema/views.py

```diff
@@ -87,6 +109,8 @@
         )
     )
     serializer_class = MovieSessionSerializer
+    authentication_classes = (TokenAuthentication,)
+    permission_classes = [IsAdminOrIfAuthenticatedReadOnly]
```

Comment:
The `MovieSessionViewSet` is configured correctly with the necessary authentication and permission classes.